include/fc/static_variant.hpp

```diff
@@ -400,36 +260,40 @@ class static_variant {
     template<typename visitor>
     static typename visitor::result_type visit( tag_type tag, visitor& v, void* data )
     {
-        static const auto wrappers = init_wrappers<impl::type_info<Types...>::count,visitor,void*,Types...>( v, data );
         FC_ASSERT( tag >= 0 && tag < count(), "Unsupported type ${tag}!", ("tag",tag) );
-        return wrappers[tag]( v, data );
+        return typelist::runtime::dispatch(list(), tag, [&v, data](auto t) {
+            return v(*reinterpret_cast<typename decltype(t)::type*>(data));
+        });
     }
 
     template<typename visitor>
     static typename visitor::result_type visit( tag_type tag, const visitor& v, void* data )
     {
-        static const auto wrappers = init_wrappers<impl::type_info<Types...>::count,const visitor,void*,Types...>( v, data );
         FC_ASSERT( tag >= 0 && tag < count(), "Unsupported type ${tag}!", ("tag",tag) );
-        return wrappers[tag]( v, data );
+        return typelist::runtime::dispatch(list(), tag, [&v, data](auto t) {
+            return v(*reinterpret_cast<typename decltype(t)::type*>(data));
+        });
     }
 
     template<typename visitor>
     static typename visitor::result_type visit( tag_type tag, visitor& v, const void* data )
     {
-        static const auto wrappers = init_const_wrappers<impl::type_info<Types...>::count,visitor,const void*,Types...>( v, data );
         FC_ASSERT( tag >= 0 && tag < count(), "Unsupported type ${tag}!", ("tag",tag) );
-        return wrappers[tag]( v, data );
+        return typelist::runtime::dispatch(list(), tag, [&v, data](auto t) {
+            return v(*reinterpret_cast<const typename decltype(t)::type*>(data));
+        });
     }
 
     template<typename visitor>
     static typename visitor::result_type visit( tag_type tag, const visitor& v, const void* data )
     {
-        static const auto wrappers = init_const_wrappers<impl::type_info<Types...>::count,const visitor,const void*,Types...>( v, data );
         FC_ASSERT( tag >= 0 && tag < count(), "Unsupported type ${tag}!", ("tag",tag) );
-        return wrappers[tag]( v, data );
+        return typelist::runtime::dispatch(list(), tag, [&v, data](auto t) {
+            return v(*reinterpret_cast<const typename decltype(t)::type*>(data));
+        });
     }
 
-    static constexpr int count() { return impl::type_info<Types...>::count; }
+    static int count() { return typelist::length<list>(); }
```

Comment:
Since typelist::length() is still `constexpr`, why `count()` is no longer `constexpr`?